made about page responsive

S-kus opened the pull request:
closes: #69 
Made about page responsive with required change in about.html and style.css.

sinc0115 commented:
Could you describe the changes you made to make it responsive? 

I see a lot of highlighting in the code diff (presumably from your code editor fixing code formatting) and it's hard to see changes made by you.

S-kus commented:
@sinc0115 to make the about page responsive, I made the following changes-

- In index.html, I replaced the paragraph ".aboutMainIngredient" with an unordered list (changes from line 253 to 268).
- In style.css, I added the required media queries to make the page adjust according to screen size (changes from line 78 to 86).

The various highlighting in the code is due to the prettier extension in my vs-code which I forget to turn off. However, It merely changed the spacing and did not mess with the code. I can still restore the code as it was previously by making a new PR if you want.

sinc0115 commented:
Your changes have not improved the responsiveness of the site, but I appreciate your interest in participating
